KS.Fiks.IO.Politisk.Behandling.Client/Models/PolitiskBehandlingMeldingTypeV1.cs

```diff
@@ -3,20 +3,22 @@ namespace KS.Fiks.IO.Politiskbehandling.Client.Models
     public class PolitiskBehandlingMeldingTypeV1
     {
         // Forespørsler
-        public const string HentMoeteplan = "no.ks.fiks.politisk.behandling.klient.hentmoeteplan.v1";
-        public const string HentUtvalg = "no.ks.fiks.politisk.behandling.klient.hentutvalg.v1";
+        public const string HentMoeteplan = "no.ks.fiks.politisk.behandling.hentmoeteplan.v1";
+        public const string HentUtvalg = "no.ks.fiks.politisk.behandling.hentutvalg.v1";
         
-        // Resultat
-        public const string ResultatMoeteplan = "no.ks.fiks.politisk.behandling.tjener.resultatmoeteplan.v1";
-        public const string ResultatUtvalg = "no.ks.fiks.politisk.behandling.tjener.resultatutvalg.v1";
-        public const string ResultatSendVedtakFraUtvalg = "no.ks.fiks.politisk.behandling.tjener.sendvedtakfrautvalg.v1";
```

Comment:
Det bør vel være et resultat her også slik at fagsystemet for utvalgsbehandling vet at svaret har kommet fram til det initierende fagsystemet. Med resultat blir alle meldinger modellert likt uavhengig av retning på meldingene.